### PR TITLE
Added tools for ignored test in .pack-ignore

### PR DESCRIPTION
**Pull request by DeanArbel**
## Status
- [x] Ready

## Related Issues
required for: https://github.com/demisto/etc/issues/25947

## Description
Added tools for ignore tests via `.pack-ignore` file.
If the test playbook is specified in `.pack-ignore` for example:
```
[file:playbook-Special-Test-Not-To-Run-Directly.yml]
ignore=auto-test
```

## Must have
- [x] Tests

### CODE REVIEW

File: demisto_sdk/commands/common/tools.py [hunk 1]
```diff
@@ -653,6 +655,39 @@ def pack_name_to_path(pack_name):
...
+
+                # create file specific ignored tests set
+                for section in config.sections():
+                    if section.startswith("file:"):
+                        file_name = section[5:]
+                        for key in config[section]:
+                            if key == 'ignore':
+                                ignore_list = str(config[section][key]).split(',')
+                                if PACK_IGNORE_TEST_FLAG in ignore_list:
+                                    path = os.path.join(PACKS_DIR, pack_name, TEST_PLAYBOOKS_DIR, file_name)
```
Comment (bakatzir) [inline demisto_sdk/commands/common/tools.py:680]: can you please add inline comments(preferable with an examples)
or even docstring with an example to the func
as it is hard to follow.

File: demisto_sdk/commands/common/tests/tools_test.py [hunk 2]
```diff
@@ -420,3 +427,137 @@ def test_origin_content(mocker, git_value, response):
...
+    # prepare repo
+    repo = Repo(tmpdir)
+    packs_path = Path(repo.path) / PACKS_DIR
+    pack = Pack(packs_path, fake_pack_name, repo)
+    test_playbook_path = packs_path / fake_pack_name / TEST_PLAYBOOKS_DIR
+    pack_ignore_path = os.path.join(pack.path, PACKS_PACK_IGNORE_FILE_NAME)
+
+    # prepare .pack-ignore
+    with open(pack_ignore_path, 'a') as pack_ignore_f:
+        pack_ignore_f.write("[file:TestIntegration.yml]\nignore=IN126\n\n"
```
Comment (bakatzir) [inline demisto_sdk/commands/common/tests/tools_test.py:555]: why is the line with IN126 is needed? (parameter_missing_from_yml_not_community_contributor)

**Review by bakatzir — COMMENTED** [review]
This is a 1st PR prior to the content PR, right?

File: demisto_sdk/commands/common/tests/tools_test.py (hunk 2 above)
Comment (DeanArbel) [reply demisto_sdk/commands/common/tests/tools_test.py:555]: It's not required, it's there for sanity checking - to make sure it doesn't create any issues

Comment (bakatzir) [reply demisto_sdk/commands/common/tests/tools_test.py:555]: legit

**Comment by DeanArbel** [comment]
> This is a 1st PR prior to the content PR, right?

Correct.

**Review by bakatzir — APPROVED** [review]
